pkgs/development/web/pnpm-lock-export/default.nix

```
@@ -1,16 +1,16 @@
 { lib, buildNpmPackage, fetchFromGitHub }:
 buildNpmPackage rec {
   pname = "pnpm-lock-export";
-  version = "0.4.0";
+  version = "0.4.0-post1";
 
   src = fetchFromGitHub {
     owner = "cvent";
```

Oops, this should be
```suggestion
    owner = "adamcstephens";
```

---

@adamcstephens will you try to open a PR for your patches?

---

Done, though I'll note it's a different branch that's a bit cleaner for a PR: https://github.com/cvent/pnpm-lock-export/pull/9